.github/ISSUE_TEMPLATE/01-bug_report.yml

```diff
@@ -1,13 +1,8 @@
 name: Bug report
 description: Report an unexpected problem
-title: 'bug: '
```

Comment:
@alexcb why was the title removed? IMO it provides more ctxt while navigating across tickets while the labels allow for easier grouping of the tickets

Reply:
ok, very well then